src/dlr_common.cc

```diff
@@ -68,7 +68,23 @@ void dlr::LoadJsonFromFile(const std::string& path, nlohmann::json& jsonObject)
   }
 }
 
-DLRBackend dlr::GetBackend(std::vector<std::string> dir_paths) {
+void dlr::LoadJsonFromString(const std::string& jsonData, nlohmann::json& jsonObject) {
+  std::stringstream jsonStrStream(jsonData);
+  try {
+    jsonStrStream >> jsonObject;
+  } catch (nlohmann::json::exception&) {
+    jsonObject = nullptr;
+  }
+}
+
+std::string dlr::LoadFileToString(const std::string& path, std::ios_base::openmode mode) {
+  std::ifstream fstream(path, mode);
+  std::stringstream blob;
+  blob << fstream.rdbuf();
+  return blob.str();
```

Comment:
@trevor-m whats the diff between doing this ^^ versus:
```
std::ifstream relay_ob(el.path, std::ios::binary);
        code_data.assign((std::istreambuf_iterator<char>(relay_ob)),
                         std::istreambuf_iterator<char>());
```
that you suggested?

Reply:
I'm not sure, it seems like it might be the same

Reply:
ok, using LoadFileToString for relay_exec ro files as well.